src/CoreBundle/private/modules/CMSRunCrons/CMSRunCrons.class.php

```diff
@@ -104,11 +139,26 @@ private function cronJobClassFactory($oTdbCmsCronJob)
         return $this->getCronjobFactory()->constructCronJob($oTdbCmsCronJob->sqlData['cron_class'], $oTdbCmsCronJob->sqlData);
     }
 
+    private function displayError(string $errorMessage): void
+    {
+        $this->data['sMessageOutput'] = sprintf('<div style="color: red">%s</div>', $errorMessage);
```

Comment:
I wasn't aware that we allow HTML here. We shouldn't do this, but instead quote message output (in a separate PR).

Reply:
I'm afraid no one will ever read such a note. Could you instead file an issue so that we think about translating the messages in TCMSCronJob (there might be reasons for the current behavior, such as scripts or monitoring systems interpreting the message)?

And as written above, I wouldn't add HTML but instead quote (= escape) message output in a separate issue and PR. HTML isn't really needed here, and it might grow into a potential security issue if we output unescaped HTML.

Reply:
ok (removed NOTE: there is no simply meaningful proposal for that location)